scripts/report_missing_files.py

```diff
@@ -0,0 +1,46 @@
+import botocore
+import requests
+import sys
+
+from managers import DBManager, S3Manager
+from model import Part, Record
+
+
+'''
+Usage:  python main.py --script ReportFileAccessibility -e <env> options source=<source>
+'''
+def main(*args):
+    db_manager = DBManager()
+    db_manager.createSession()
+
+    s3_manager = S3Manager()
+    s3_manager.createS3Client()
+
+    source_arg = next(filter(lambda option: option.startswith('source'), args), None)
```

Comment:
kinda subjective, but comprehensions are *generally* considered more pythonic:

```python
source_args = next((option for option in args if option.startswith("source"), None)
```

Reply:
just learned actually you can do generator comps too!